src/Aquifer.Public.API/Endpoints/Resources/Updates/List/Validator.cs

```diff
@@ -9,5 +9,13 @@ public Validator()
     {
         RuleFor(x => x.Timestamp).NotEmpty().GreaterThanOrEqualTo(DateTime.UtcNow.AddDays(-90).Date);
         RuleFor(x => x.LanguageId).InclusiveBetween(1, 32);
+        RuleFor(x => x.Offset).GreaterThanOrEqualTo(0);
+        RuleFor(x => x.Limit).InclusiveBetween(0, 1000);
```

Comment:
I assume Limit 0 would just return no results, which I don't think makes sense. Probably should make this at least 1 to 1000.